cmd/image/import/import.go

```diff
@@ -182,70 +184,71 @@ pvsadm image import -n upstream-core-lon04 -b <BUCKETNAME>  --object rhel-83-100
 		if opt.Public {
 			bucketAccess = "public"
 		}
+		klog.Infof("Importing image %s. Please wait...", opt.ImageName)
 		jobRef, err := pvmclient.ImgClient.ImportImage(opt.ImageName, opt.ImageFilename, opt.Region,
 			opt.AccessKey, opt.SecretKey, opt.BucketName, strings.ToLower(opt.StorageType), bucketAccess)
 		if err != nil {
 			return err
 		}
 
+		timeOut := time.After(opt.WatchTimeout)
+		pollInterval := time.Tick(2 * time.Minute)
+
 		start := time.Now()
-		pollErr := wait.PollImmediate(2*time.Minute, opt.WatchTimeout, func() (bool, error) {
-			job, err := pvmclient.JobClient.Get(*jobRef.ID)
-			if err != nil {
-				return false, err
-			}
-			if *job.Status.State == "completed" {
-				return true, nil
-			}
-			if *job.Status.State == "failed" {
-				return false, fmt.Errorf("image import job failed to complete, err: %v", job.Status.Message)
+		for done := false; !done; {
```

Comment:
wondering if its worth creating this poll logic into a util function and use that instead?

Reply:
Agree. This is now achieved through `WaitUntilConditionIsMet`, defined in the utils package, similar to the previous functionality achieved through `wait.PollImmediate`.